blaze/expr/datetime.py

```diff
@@ -222,10 +246,149 @@ def truncate(expr, *args, **kwargs):
+    _arguments = '_child'
+
+
+class Round(ElemWise):
+    _arguments = '_child', 'freq'
+
+    @property
+    def schema(self):
+        return self._child.schema
+
+
+class Floor(ElemWise):
+    _arguments = '_child', 'freq'
+
+    @property
+    def schema(self):
+        return self._child.schema
+
+
+class Ceil(ElemWise):
+    _arguments = '_child', 'freq'
+
+    @property
+    def schema(self):
+        return self._child.schema
+
+
+class dt_ns(object):
+
+    def __init__(self, field):
+        self.field = field
+
+    def year(self):
+        return year(self.field)
+    def month(self):
+        return month(self.field)
+    def day(self):
+        return day(self.field)
+    def hour(self):
+        return hour(self.field)
+    def minute(self):
+        return minute(self.field)
+    def date(self):
+        return date(self.field)
+    def time(self):
+        return time(self.field)
+    def second(self):
+        return second(self.field)
+    def millisecond(self):
+        return millisecond(self.field)
+    def microsecond(self):
+        return microsecond(self.field)
+    def nanosecond(self):
+        return nanosecond(self.field)
+    def weekday(self):
+        return weekday(self.field)
+    def weekday_name(self):
+        return weekday_name(self.field)
+    def daysinmonth(self):
+        return daysinmonth(self.field)
+    def weekofyear(self):
+        return weekofyear(self.field)
+    def dayofyear(self):
+        return dayofyear(self.field)
+    def dayofweek(self):
+        return dayofweek(self.field)
+    def quarter(self):
+        return quarter(self.field)
+    def is_month_start(self):
+        return is_month_start(self.field)
+    def is_month_end(self):
+        return is_month_end(self.field)
+    def is_quarter_start(self):
+        return is_quarter_start(self.field)
+    def is_quarter_end(self):
+        return is_quarter_end(self.field)
+    def is_year_start(self):
+        return is_year_start(self.field)
+    def is_year_end(self):
+        return is_year_end(self.field)
+    def days_in_month(self):
+        return days_in_month(self.field)
+    def strftime(self, format):
+        return strftime(self.field, format)
+    def truncate(self, *args, **kwargs):
+        return truncate(self.field, *args, **kwargs)
+    def round(self, freq):
+        return Round(self.field, freq)
+    def floor(self, freq):
+        return Floor(self.field, freq)
+    def ceil(self, freq):
+        return Ceil(self.field, freq)
+    def week(self):
+        return week(self.field)
+
+class dt(object):
+
+    __name__ = 'dt'
+
+    def __get__(self, obj, type=None):
+        return dt_ns(obj) if obj is not None else self
+
+
+class days(DateTime): _dtype = datashape.int64
+class nanoseconds(DateTime): _dtype = datashape.int64
+class seconds(DateTime): _dtype = datashape.int64
+class total_seconds(DateTime): _dtype = datashape.int64
+
+
+class td_ns(object):
```

Comment:
Can we rename this to something like `timedelta_ns`?  Using `td` as the abbreviation isn't obvious at first glance.